ui/jstests/test_lr_exports.jsx

```diff
@@ -824,7 +824,7 @@ define(['QUnit', 'jquery', 'lr_exports', 'reactaddons',
 
     assert.equal(0, $(container).find("div").size());
     Exports.loader("repo", "user", function() {}, container);
-    assert.equal(1, $(container).find("div").size());
+    assert.equal(2, $(container).find("div").size());
```

Comment:
Because now there will be always a message <div> in panel with no text, That , made div size 2.

see react class StatusBox